### PR TITLE
feat: added ntfy provider

### LINKED ISSUE #1122: ntfy.sh Provider
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
https://docs.ntfy.sh/publish/

**Additional context**
Integration of ntfy.sh project for send alert

### PR DESCRIPTION
**Pull request by ezhil56x**
/claim #1122
Closes #1122

## 📑 Description
Added Ntfy.sh provider for sending alert

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by ezhil56x** [comment]
https://github.com/keephq/keep/assets/103899034/cd77e584-c120-4e76-b3bd-55391a9fcbe6

File: README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@
 <h1 align="center">The open-source alerts management and automation platform</h1>
```
Comment (shahargl) [inline README.md:5]: please revert all README changes and just add the icon to the providers list

File: docs/providers/documentation/ntfy-provider.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Login to your Ntfy.sh account.
+2. Click on `Subscribe to a topic` button and generate name for the topic and subscribe to it.
+3. Copy the generated topic name. This will be used as the `Ntfy Subcription Topic` in the provider settings.
+4. Reserve the topic and confiure access (Requires ntfy Pro)
+
+## Usefull Links
+
+- [Ntfy.sh](https://ntfy.sh/)
+- [Ntfy.sh Pricing](https://ntfy.sh/#pricing)
```
Comment (shahargl) [inline docs/providers/documentation/ntfy-provider.mdx:33]: remove pricing

File: docs/providers/documentation/ntfy-provider.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (shahargl) [inline docs/providers/documentation/ntfy-provider.mdx:1]: add reference to `mint.json`

**Comment by ezhil56x** [comment]
@shahargl 
Updated my PR. Now it works with self hosted ntfy as well. And made the necessary changes. Can I get a review once?
Thanks!

**Pull request merged**